roxctl/central/generate/generate.go

```diff
@@ -255,7 +256,7 @@ func createBundle(config *renderer.Config) (*zip.Wrapper, error) {
 
 // OutputZip renders a deployment bundle. The deployment bundle can either be
 // written directly into a directory, or as a zipfile to STDOUT.
-func OutputZip(logger logger.Logger, io io2.IO, config renderer.Config) error {
+func OutputZip(logger logger.Logger, io io2.IO, config renderer.Config, flags *pflag.FlagSet) error {
```

Comment:
it's used only in this package
```suggestion
func outputZip(logger logger.Logger, io io2.IO, config renderer.Config, flags *pflag.FlagSet) error {
```

Reply:
Done

Reply:
I had the strangest test failure and it turns out it's because I unexported `OutputZip`. It must be capitalized, this is checked [here](https://github.com/stackrox/stackrox/blob/868944e0b60a08ef6b5065ad51aa4436c1b5a2a5/roxctl/maincommand/command_test.go#L32-L35) in TestCommand.

Now I'm not sure _why_ this must be done, maybe it has something to do with how cobra commands are used.

Reply:
Disregard this, just looked at it in our team code review and it was actually the missing period that caused this. This coincided with my merge conflict which was from Michael having added those checks and the period at the end, hence the confusion.